.vib/appsmith/goss/vars.yaml

```diff
@@ -0,0 +1,27 @@
+binaries:
+  - java
+  - mongosh
+  - nginx
+  - node
+  - python
+  - render-template
+  - wait-for-port
+directories:
```

Comment:
Shouldn't we also check for files and directories added by the `appsmith` package?

E.g. `backend/server.jar`, `editor`, `rts/package.json`, `templates`, `templates/client`.

Reply:
There were a couple of files I missed checking, added those now. Thanks!